opengrok-indexer/src/main/java/org/opengrok/indexer/web/Util.java

```diff
@@ -770,6 +763,9 @@ public static void readableLine(int num, Writer out, Annotation annotation,
             }
             StringBuilder buf = new StringBuilder();
             htmlize(r, buf);
+            if (annotation.getFileVersion(r) == annotation.getRevisions().size()) {
```

Comment:
add comment

Reply:
Also, I wonder if this widening could cause some UI problems ? Is there perhaps better way to mark the latest revision ? (bold font, different font color, etc.)

Reply:
IDEA uses that together with bold font.  I can add a +1 to widestRevision to avoid any UI problems. Adding asterisk or any other mark seems fine to me to mark the revision.

Reply:
I'd prefer the bold font too.

Reply:
ok

Reply:
And this hasn’t been fully addressed

Reply:
done